### DISCUSSION TITLE
建议增加输入框增强的独立配置

### DISCUSSION THREAD

**Discussion by wang20150419** [immersive-translate/immersive-translate#1221] [Ideas]
### 新功能 | New Feature

输入框增强 针对具体网站进行独立配置，而不一定要跟随 immersive-translate 的全局配置。

### 为什么您想要添加这些功能 | Why you want to add these features

具体而言，很多网站，只需要 使用 输入框增强 功能，而不是针对该网站的全局翻译功能。

使用场景：

搜索谷歌，并不想用  immersive-translate，因为这样会有大量的无效翻译（搜索返回的结果，很多是不需要的）。
很多情况下，把搜索的关键词，翻译为英文后进行搜索，结果会更好一些。这个时候，如果 能使用 输入框增强 增强功能，就非常方便，不需要切换了。

**Comment by theowenyoung** [comment]
本身就是独立的呀，难道你使用输入增强之后，页面会自动翻译？
